build.gradle.kts

```diff
@@ -17,15 +17,32 @@ liquibase {
         register("main") {
             arguments = mapOf(
                 "changeLogFile" to "db/db.changelog-master.xml",
-                "url" to databaseCredentials.databaseUrl,
+                "url" to "${ databaseCredentials.databaseUrl}?createDatabaseIfNotExist=true",
                 "username" to databaseCredentials.username,
                 "password" to databaseCredentials.password,
                 "logLevel" to "info",
                 "contexts" to when (profile) {
                     "prod" -> "prod"
                     "dev" -> "dev"
                     else -> throw GradleException("Profile $profile not configured to map on a particular liquibase context")
-                }
+                },
+                "liquibaseSchemaName" to "save_cloud"
+            )
+        }
+        register("sandbox") {
+            arguments = mapOf(
+                "changeLogFile" to "save-sandbox/db/db.changelog-sandbox.xml",
+                "url" to "${ databaseCredentials.databaseUrl}?createDatabaseIfNotExist=true",
```

Comment:
I think `databaseCredentials.databaseUrl` here should be changed to `databaseCredentials.sandboxDatabaseUrl` or similar.
should be another databaseUrl for sandbox